tests/src/integration/common/wskdeploy.go

```diff
@@ -47,17 +47,17 @@ func (wskdeploy *Wskdeploy) RunCommand(s ...string) ([]byte, error) {
 }
 
 func (wskdeploy *Wskdeploy) Deploy(manifestPath string, deploymentPath string) ([]byte, error) {
-	return wskdeploy.RunCommand("-m", manifestPath, "-d", deploymentPath)
+	return wskdeploy.RunCommand("-i ", "false", "-m", manifestPath, "-d", deploymentPath)
```

Comment:
Just merged https://github.com/apache/incubator-openwhisk-wskdeploy/pull/375, we now have the flag "UserInteractive" (default to false), but need to verify on this issue if it had the desired effect.